data/saas/config/mixpanel_config.yml

```diff
@@ -0,0 +1,120 @@
+saas_config:
+  fides_key: mixpanel_connector_example
+  name: Mixpanel SaaS Config
+  description: A sample schema representing the Mixpanel connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: gdpr_oauth_token
+    - name: domain
+    - name: api_domain
+    - name: username
+    - name: password
+    - name: project_id
+
+  client_config:
+    protocol: https
+    host:
+      connector_param: domain
+    authentication:
+      strategy: basic_authentication
+      configuration:
+        username:
+          connector_param: username
+        password:
+          connector_param: password
+
+  test_request:
+    path: /api/2.0/annotations
+    method: GET
+
+  endpoints:
+  - name: user
+    requests:
+      read:
+        path: /api/2.0/engage/
+        method: GET
+        query_params: 
+          - name: project_id
+            value: <project_id>
+        body: '{
+            "where": "properties[\"$email\"] == \"<email>\""
+          }'
+        param_values:
+          - name: email
+            identity: email
+            data_type: string
+          - name: project_id
+            connector_param: project_id
+        data_path: results
+      update:
+        path: /engage#profile-set
+        method: POST
+        body: '[{
+            "$token": "<token>",
+            "$distinct_id": "<distinct_id>",
+          }]'
+        param_values:
+          - name: token
+            connector_param: token
+          - name: distinct_id
+            references:
+              - dataset: mixpanel_connector_example
+                field: user.$distinct_id
+                direction: from
+        client_config:
+          protocol: https
+          host:
+            connector_param: api_domain
+          authentication:
+            strategy: basic_authentication
+            configuration:
+              username:
+                connector_param: username
+              password:
+                connector_param: password
+      delete:
```

Comment:
This `delete` request should be rewritten as `data_protection_request` since it calls Mixpanel's dedicated GDPR endpoint.
https://ethyca.github.io/fidesops/guides/saas_config/#data-protection-request